README.md

```diff
@@ -13,6 +13,8 @@ React components for [Stripe.js](https://stripe.com/docs/stripe-js) and
 
 ## Documentation
 
+### We’re currently beta-testing improved documentation for this new library. Please contact us at stripejs-feedback@stripe.com and provide your Stripe account ID to get access to the new documentation!
```

Comment:
```suggestion
We’re currently beta-testing this library. Please contact us at stripejs-feedback@stripe.com and provide your Stripe account ID for access to new documentation!
```

Reply:
Um we should point out that it's for documentation on stripe.com